docker/Dockerfile

```diff
@@ -0,0 +1,47 @@
+# This file creates a Docker image for running FITS within Tomcat.
+#
+# Usage:
+#   # Build the image
+#   mvn -DskipTests clean package
+#   mvn dependency:copy-dependencies -DincludeArtifactIds=fits -DincludeTypes=zip
+#   docker build -f docker/Dockerfile -t fitsservlet .
+#
+#   # Run the service in the foreground
+#   docker run --rm -it -p 8080:8080 fitsservlet
+#
+#   # Navigate to http://localhost:8080 in your browser
+
+FROM docker.io/tomcat:9-jre17-temurin-focal
+
+ARG FILE_VERSION=5.41
+
+RUN apt-get update && \
+    apt-get install -yqq make gcc unzip && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
Could we add a couple of text editors to modify the fits.xml file (e.g. turn on tool output)?
```
    apt-get install nano -yqq && \
    apt-get install vim -yqq && \

```